### ISSUE TITLE
Display the progress bar for macOS doc refresh

### ISSUE THREAD

**Issue by adomasven** [zotero/zotero#1923]
This is requested every now and then, and we should at least reconsider it, since it creates quite an inconvenience for people with hour-long refreshes https://forums.zotero.org/discussion/86458/feature-request-refresh-progress-indicator#latest.

This is the original reasoning for why we chose to not show it: 
https://github.com/zotero/zotero/blob/8fc316f72796d8b7ff4e29e5f8bc3123d8d2a7e7/chrome/content/zotero/xpcom/integration.js#L3030-L3039

**Comment by dstillman** [comment]
Does this mean we do show a progress meter for word processor inserts/refreshes on Windows/Linux?

**Comment by adomasven** [comment]
> Does this mean we do show a progress meter for word processor inserts/refreshes on Windows/Linux?

Yep. I remember we discussed this decision at length when I was adding the progress bar (used to be no progress bar for refreshes on all platforms).

Also the original reasoning for Google Docs does not apply since we show the progress overlay there (initially we didn't have one).

**Comment by adomasven** [comment]
I found some of the discussion here: https://github.com/zotero/zotero/pull/1476#discussion_r183214774

**Comment by gagarine** [comment]
Progress indicator (already just activity indicator actually like a animated spinner) would be super nice. Yet changing the focus from Word to Zotero is disturbing for the user. Would be nice to have this indicator in Word, if possible by animating the icon for example.
